build/cleanup-images.yml

```diff
@@ -4,45 +4,26 @@
 variables:
 - template: build-variables.yml
 
-jobs:
-- job: deleteImage
-  displayName: 'Delete Images'
-  pool:
-    vmImage: $(WindowsVmImage)
-  steps:
-  - task: AzureCLI@2
-    displayName: 'Delete Images'
-    inputs:
-      azureSubscription: $(ConnectedServiceName)
-      scriptType: ps
-      scriptLocation: InlineScript
-      inlineScript: |
-        $versions = "stu3","r4","r5"
-        
-        foreach ($version in $versions) {
-          $repository = "${version}_fhir-server"
-          $tagList = az acr repository show-tags --name $(azureContainerRegistryName) --repository $repository | ConvertFrom-Json
+stages:
+- stage: CleanupStu3
+  displayName: 'Cleanup Stu3'
+  jobs:
+  - template: ./jobs/cleanup-version-images.yml
+    parameters: 
+      version: 'stu3'
```

Comment:
>stu3 [](start = 16, length = 4)

nit: does the case matter here? In other yml files we seem to pass in Stu3, R4 and R5 for the version parameters.

Reply:
In this case it does because it has to match the image name, which I'm pretty sure is case sensitive. We use the uppercase ones when it is display or documentation and lowercase for code uses.